packages/core/src/components/validation-tooltip/validation-tooltip.tsx

```diff
@@ -187,7 +189,7 @@ export class ValidationTooltip {
     this.observer?.disconnect();
     this.destroyAutoUpdate();
 
-    this.formElement.removeEventListener('submit', this.onSubmitBind);
+    this.formElement?.removeEventListener('submit', this.onSubmitBind);
     this.inputElement.removeEventListener('focus', this.onInputFocusBind);
```

Comment:
Should we not also add a null check to this line?

Reply:
I wasn't sure if the IxValidationTooltip can be used without an input field. If it can a null check should be added, since this might lead to an error otherwise.

Reply:
I think the problem is that in some test setups timing issues can lead NPEs even if an input was at some point present.